frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -23,6 +23,11 @@ CELLxGENE is focused on supporting the global community attempting to create ref
 - cell lines
 - organisms other than mouse or human
 
+### Scale Requirements
+
+- CELLxGENE Discover sets the maximum dataset size for uploads to 30GB. This is enforced in the UX by a property in the [Dropbox Chooser](https://github.com/chanzuckerberg/single-cell-data-portal/blob/75c648e8b277d23c4ff0fe198754e660dfd981d9/frontend/src/components/DropboxChooser/index.tsx#L29). There is pending discovery to support larger dataset sizes.
```

Comment:
I would rewrite to:

> CELLxGENE Discover sets the maximum dataset size to 50GB.

There is a 30 GB limitation in the Curation UX (enforced by the dropbox chooser) and a 50 GB limitation in the Discover API, but both cases are implementation details that apply to curators and should not be generally documented. @Bento007 is a good contact about the details if you're curious. 

Also, here's what Lattice shares with contributors in their "first contact" emails:

**Scale limitations**

H5AD files must be less than 50 GB for submission

Datasets with more than 4.3M observations will not have visualizations available, but data will be accessible via download and in CELLxGENE features, such as Census

Datasets with more than 2^31 -1 non-zero elements in a single data layer will not have RDS files available (per R dgCMatrix sparse matrix class limitations), but data will be accessible via visualization, H5AD download and in CELLxGENE features, such as Census

Reply:
Thanks for the context! Updated.